runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -346,10 +356,8 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 		}
 
 		GameObject gameObject = event.getGameObject();
-		if (ROCK_OBSTACLES.contains(gameObject.getId()))
-		{
-			rocks.add(gameObject);
-		}
+
+		addGameObject(gameObject);
```

Comment:
`gameObject` is only used once now. Change this to

```java
addGameObject(event.getGameObject());
```

Reply:
done